### ISSUE TITLE
Longhorn Beetle Mutants

### ISSUE THREAD

**Issue by sadenar** [CleverRaven/Cataclysm-DDA#65285]
### Is your feature request related to a problem? Please describe.

Somebody on the devcord (Khorne Flakes#3162) proposed a new idea for a mutant line, (Asian) longhorn beetles that I thought would be interesting and reasonable to include in the game because:

a) Beetles aren't an overly crowded mutant type right now, only containing diving and stag beetles, so no concern of overcrowded monsters like adding YET another flavor of zombie or feral (no offense to those who do add them, but it's hard to not make some enemies same-y at some point when you add 50 of them)

b) Longhorn beetle mutants can have a unique flavor applied to them, their relative aggressiveness for a beetle could be tuned up by mutagen influence, they are apparently quite resilient for an insect, they are somewhat slow, and you can't tell me those huge antennae just beg to be mutagen'd into grasping appendages/whips:
![anogl11](https://user-images.githubusercontent.com/16321433/233874566-1b5fc42e-4a81-475f-9f22-9f5c6a4b2d9e.png)

c) Contrary to most "cool" additions, longhorn beetles are invasive yet present IRL in NE, as shown by this map I found:
![217802](https://user-images.githubusercontent.com/16321433/233874621-588a283a-5b03-4c32-9172-674daa2bd116.jpg)
So lore wise it is not as much of a stretch to include them as it is to include more exotic creatures, as despite being invasive, it is reasonable to expect the little critters to have drank enough of the wild mutagen juice to have mutated just like all the other mutated critters.

### Solution you would like.

So to sum it up, this new insect mutant line would be a twist on the stag beetle recipe, the special stag_smash attack being removed either with a short range whip/grab attack (maybe no special first stage, whip 2nd stage, grasp on 3rd?) and additional balance could probably include less speed than the stag beetle.

### Describe alternatives you have considered.

Not asking for the addition of a cool new critter of a type of insect that's been maybe somewhat neglected by the wonders of mutation?

### Additional context

Beetles!
